src/main/java/org/retest/rebazer/service/BitbucketService.java

```diff
@@ -52,32 +52,34 @@ public BitbucketService(RebaseService rebaseService) {
 
 	@Scheduled(fixedDelay = 10 * 1000)
 	public void pollBitbucket() {
-		config.getRepos().forEach(repo -> {
-			log.info("Processing repository: {}", repo.getName());
-			getAllPullRequestIds(repo).forEach(pullRequest -> handlePR(repo, pullRequest));
-		});
+		for (Repository repo : config.getRepos()) {
+			log.debug("Processing {}.", repo);
```

Comment:
Id prefer log.info("Processing repository {}.", repo.getName());

Reply:
Personally, I'm not sure which log level is the best for us. If we choose `INFO`, we get a log entry for each PR of each repo every 10s. That might be quite a lot. However, I'll revert to `INFO` and we'll see how it goes.

Reply:
Regarding the actual log message: According to Bloch's Effective Java, "the `toString` method should return *all* of the interesting information contained in the object". Therefore, I think we should adapt the corresponding `toString` methods, rather than constructing hand-crafted log messages every time we use these objects.